github_test.sh

```diff
@@ -37,4 +37,4 @@ fi
 SLANG_TEST=${OUTPUTDIR}slang-test
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$OUTPUTDIR
-${SLANG_TEST} -bindir ${OUTPUTDIR} -travis -category ${SLANG_TEST_CATEGORY} ${SLANG_TEST_FLAGS}
+${SLANG_TEST} -bindir ${OUTPUTDIR} -travis -category ${SLANG_TEST_CATEGORY} ${SLANG_TEST_FLAGS} -api all-vk
```

Comment:
Presumably this is because of the issues with CI/Vk, so this PR disables those?

Reply:
Right, the swiftshader shared library on linux suddently stops working, as long as a vulkan call is made, it ran into a `Multithreading must be enabled to use std::thread()" error, seems like some pthread linking issue that wasn't happening before.

I suspect this is related to some system update of the CI image.